src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -82,27 +97,48 @@ const CrossPodQueryForm = () => {
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleCrossPodQuery} autoComplete="off">
+      <Box sx={{ minWidth: 120 }}>
         <div style={formRowStyle}>
-          <label htmlFor="cross-search-doc">Search document from username: </label>
-          <br />
-          <br />
-          <input
+          <TextField
+            {...username}
+            label="Search document from username"
+            InputProps={{
+              type: 'search'
+            }}
             id="cross-search-doc"
-            size="60"
+            fullWidth
             name="crossPodQuery"
-            {...username}
             placeholder={selectedUser}
+            value={searchUser}
```

Comment:
Related to Line 32, we already have a state that grabs our username. Line 29 also contains the onChange function within {...username}. Revert Line 112 to 115 back to {...username}.